Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM gradle:jdk17-alpine AS build
+# copy source to image
+WORKDIR /app
+COPY . /app
+ENV GRADLE_OPTS="-Dorg.gradle.daemon=false"
+# build jar by gradle
+RUN gradle build -x test
```

Comment:
If you want to use application-docker.yml, I think there is a setting like this below
![image](https://github.com/user-attachments/assets/2a3987f5-fd4e-4f8e-8fbb-3fc0fe781e6e)